### PR TITLE
Update cargos

### PR DESCRIPTION
**Pull request by crocme10**
Change edition 2018 -> 2021
Simplify authors
Update some crates (tokio, ...)

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -27,6 +27,14 @@ jobs:
...
         with:
           fetch-depth: 0
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+
+      - name: Check versions
+        run: |
+          rustc --version
+          cargo --version
```
Comment (remi-dupre) [inline .github/workflows/docker.yml:37]: As this is unrelated to the Rust version used by the docker builder, we probably don't need to add it here?

File: Cargo.toml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 [package]
 name = "mimirsbrunn"
 version = "2.0.0-rc1"
-authors = ["Kisio Digital <team.core@kisio.com>", "Qwant", "Guillaume Pinot <texitoi@texitio.eu>"]
+authors = ["Kisio Digital", "Qwant"]
```
Comment (remi-dupre) [inline Cargo.toml:4]: Thanks! :smile:

File: .github/workflows/docker.yml (hunk 1 above)
Comment (crocme10) [reply .github/workflows/docker.yml:37]: yes, agreed, and removed.

It would be nice to use the version of rust found in rust-toolchain in the dockerfile

**Review by remi-dupre — APPROVED** [review]
